browser/brave_stats/brave_stats_updater_browsertest.cc

```diff
@@ -251,3 +233,30 @@ IN_PROC_BROWSER_TEST_F(BraveStatsUpdaterReferralCodeBrowserTest,
   EXPECT_TRUE(net::GetValueForKeyInQuery(update_url, "ref", &query_value));
   EXPECT_EQ(query_value, referral_code());
 }
+
+#if !BUILDFLAG(IS_ANDROID)
+class BraveStatsUpdaterDisabledBrowserTest
+    : public BraveStatsUpdaterBrowserTest {
+ public:
+  void SetUp() override {
+    brave_stats::BraveStatsUpdater::DisableAutoStartForTesting();
```

Comment:
Ideally rather than just disabling on `SetUp()` it is good also to re-enable on `TearDown()`.